platforms/Linux/README.md

```diff
@@ -1,3 +1,100 @@
+* Package name: swiftlang
+* License: Apache 2.0
+* Maintainer: swift-infrastructure@forums.swift.org
+* URL: https://swift.org
+* Description:
+```
+Swift is a general-purpose programming language built using
+a modern approach to safety, performance, and software design
+patterns.
+
+The goal of the Swift project is to create the best available
+language for uses ranging from systems programming, to mobile
+and desktop apps, scaling up to cloud services. Most
+importantly, Swift is designed to make writing and maintaining
+correct programs easier for the developer.
+```
+
+### RPM Naming Convention:
+
+Package naming convention: `swiftlang-<VERSION>-<RELEASE>.<DIST>.<ARCH>.rpm`  
+Package structure: `/repo/<OS>/releases/<OS_VERSION>/<ARCH>/`  
+Repository configuration: `/repo/<OS>/releases/<OS_VERSION>/swiftlang.repo`  
+
+
+#### Example
+
+* **Package structure:**  
+```
+/repo/centos/releases/8/x86_64/swiftlang-5.5.0-1.el8.x86_64.rpm
+/repo/centos/releases/8/aarch64/swiftlang-5.5.0-1.el8.aarch64.rpm
+```
+
+* **Package URL:**  
+https://download.swift.org/repo/centos/releases/8/aarch64/swiftlang-5.5.0-1.el8.aarch64.rpm
+
+* **Repository configuration file URL:**  
+https://download.swift.org/repo/centos/releases/8/swiftlang.repo
+
+## Tasks
+
+### RPM Package Manager (RPM)*
+
+- [ ] [SR-15325](https://bugs.swift.org/browse/SR-15325) Create RPM spec file
+- [ ] [SR-15326](https://bugs.swift.org/browse/SR-15326) Setup CI job to build the rpm package
+- [ ] [SR-15327](https://bugs.swift.org/browse/SR-15327) Code sign rpm package with swift.org certificate
+- [ ] [SR-15328](https://bugs.swift.org/browse/SR-15328) Host the rpm package on swift.org
+- [ ] [SR-15329](https://bugs.swift.org/browse/SR-15329) Host the rpm repository on swift.org
+- [ ] [SR-15330](https://bugs.swift.org/browse/SR-15330) Verify the rpm package and repository
+- [ ] [SR-15331](https://bugs.swift.org/browse/SR-15331) Update swift.org download / install page
+- [ ] [SR-15332](https://bugs.swift.org/browse/SR-15332) Work with official repositories to accept package specs
+
+*For each platform, we will start with CentOS 8.
+
+### Debian Package (Deb)*
+
+- [ ] [SR-15334](https://bugs.swift.org/browse/SR-15334) Create Debs control file
+- [ ] [SR-15335](https://bugs.swift.org/browse/SR-15335) Setup CI to build the deb package
+- [ ] [SR-15336](https://bugs.swift.org/browse/SR-15336) Code sign package with swift.org certificate
+- [ ] [SR-15337](https://bugs.swift.org/browse/SR-15337) Host the deb package on swift.org
+- [ ] [SR-15338](https://bugs.swift.org/browse/SR-15338) Host the deb repository on swift.org
+- [ ] [SR-15339](https://bugs.swift.org/browse/SR-15339) Verify the deb package and repository
+- [ ] [SR-15340](https://bugs.swift.org/browse/SR-15340) Update swift.org download / install page
+- [ ] [SR-15341](https://bugs.swift.org/browse/SR-15341) Work with official repositories to accept package control files
+
+*For each platform, we will start with Ubuntu 20.04
```

Comment:
_Will we?_

Reply:
🤷‍♂️ This is just movement for now. Someone should go through the content. I suggest, as the Linux Release Manager, @shahmishal be given the honour of going through the content 😆